__init__.py

```diff
@@ -112,25 +113,30 @@ def _create_spoken_stardate(self):
     def handle_pod_intent(self, _):
         self.speak_dialog("pod")
 
-    @intent_handler(
-        IntentBuilder("robotics_laws_intent")
-        .require("robotics_keyword")
-        .require("law_keyword")
-        .optionally("LawOfRobotics")
-        .build()
-    )
-    def handle_robotic_laws_intent(self, message):
-        law = str(message.data.get("LawOfRobotics", "all"))
+    @intent_handler("law_of_robotics.intent")
+    def handle_robotic_laws_intent(self, message: Message):
+        law = str(message.data.get("ordinal", ""))
+        law = extract_number(law, ordinals=True)
+        self.log.debug("law: %s", law)
+        if not law:
+            self.log.debug("No specific law detected, reciting all three")
+            self.speak_dialog("rule1")
+            self.speak_dialog("rule2")
+            self.speak_dialog("rule3")
```

Comment:
needs to return, or will trigger `  self.speak_dialog("invalid_law")` some lines below